egs/babel_multilang/s5/local/nnet3/run_ivector_common_langs.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+. ./cmd.sh
+set -e
+stage=1
+train_stage=-10
+generate_alignments=true # false if doing ctc training
+speed_perturb=true
+pitch_conf=conf/pitch.conf
+use_flp=false
+voicing_conf=
+global_extractor=exp/multi/nnet3/extractor
+ivector_suffix=
+
+[ ! -f ./conf/common_vars.sh ] && echo 'the file conf/common_vars.sh does not exist!' && exit 1
+
+. conf/common_vars.sh || exit 1;
+
+[ -f local.conf ] && . ./local.conf
+
+. ./utils/parse_options.sh
+
+L=$1
+
+mkdir -p nnet3
+# perturbed data preparation
+train_set=train
+if [ "$speed_perturb" == "true" ]; then
+  train_set=train_sp
+fi
+
+extractor=$global_extractor
```

Comment:
Why is the `extractor` variable even required ? Could you not just use `global_extractor` at line 45.